packages/terra-hookshot/src/HookshotContent.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import onClickOutside from 'react-onclickoutside';
+import styles from './HookshotContent.scss';
+
+const cx = classNames.bind(styles);
+/**
+ * Key code value for the escape key.
+ */
+const KEYCODES = {
+  ESCAPE: 27,
+};
+
+const propTypes = {
+  /**
+   * The children to be displayed as content within the content.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * The function callback when am escape keydown event occurs..
```

Comment:
You have an extra period here.

![https://i.imgur.com/u69Mljl.gif](https://i.imgur.com/u69Mljl.gif)

Reply:
typo: `am` => `an`
![wink](https://user-images.githubusercontent.com/633148/31403727-a2377ce4-adbf-11e7-87a8-0848e6ad1006.gif)